package/tests/test_server.py

```diff
@@ -49,6 +43,150 @@
 ]
 
 
+EXPECTED_PIPELINE_DATA = {
```

Comment:
Is it possible to come up with a more minimalistic test case in order to be easier to change later?